components/collectionList/CollectionItem.qml

```diff
@@ -72,6 +73,17 @@ Item {
             bold: true;
         }
     }
+	
+	SortFilterProxyModel {
+        id: sortedGamesCollection;
+
+        sourceModel: allCollections[collectionListView.currentIndex].games;
+        filters: [
+            ValueFilter { roleName: 'favorite'; value: true; enabled: onlyFavorites; },
+            ExpressionFilter { enabled: onlyMultiplayer; expression: { return players > 1; } },
+            RegExpFilter { roleName: 'title'; pattern: nameFilter; caseSensitivity: Qt.CaseInsensitive; enabled: nameFilter !== ''; }
+        ]
+    }
```

Comment:
out of all of these changes, this one is the one I'm most hesitant to merge.

I've had feedback that the collection list stutters when sorting/filtering large collections whenever the `currentIndex` changes, so I moved that process to sort/filter only when they go into the game list. I'm worried that adding filtering back in here will just reintroduce the stuttering.

On the other hand, filtering is much less intensive than sorting, so this may be fine. I'll merge this when the other changes are ready, then package a pre-release and send it to some testers before cutting a new release on the releases page.

Reply:
That's a reasonable concern. I tested it with my SNES and NES libraries of ~800 games each on my AYN Odin Pro without seeing any issues. If there is a potential issue, though, this could always be wrapped in another config flag to choose whether to show filtered count or full count.